ixmp/__init__.py

```diff
@@ -49,5 +49,6 @@
 # Configure the 'ixmp' logger: write messages to stdout, defaulting to level WARNING
 # and above
 log = logging.getLogger(__name__)
-log.addHandler(logging.StreamHandler())
-log.setLevel(logging.WARNING)
+sh = logging.StreamHandler()
+sh.setLevel(logging.WARNING)
+log.addHandler(sh)
```

Comment:
Currently, the log-level seems to be set to "DEBUG", based on the amount of on-screen logs printed.

Reply:
Reverted this commit.

The configuration for logging in ixmp is a little less clean than in `message-ix-models` etc, but a full revamp is out-of-scope for this PR.